sycl/include/CL/sycl/detail/type_traits.hpp

```diff
@@ -31,6 +46,9 @@ using half = detail::half_impl::half;
 // Forward declaration
 template <typename ElementType, access::address_space Space> class multi_ptr;
 
+template <class T>
+inline constexpr bool is_group_v = detail::is_group<T>::value;
+
```

Comment:
Please guard this with C++ 17 macro

Reply:
good catch, thanks. Turns out we have one: `__SYCL_INLINE_CONSTEXPR`